meshroom/ui/qml/GraphEditor/Node.qml

```diff
@@ -328,22 +328,23 @@ Item {
                             MaterialLabel {
                                 id: nodeImageOutput
                                 visible: (node.hasImageOutput || node.has3DOutput)
-                                         && ((["SUCCESS"].includes(node.globalStatus) && node.chunks.count > 0) || !node.isComputable)
                                 text: MaterialIcons.visibility
                                 padding: 2
                                 font.pointSize: 7
+                                property bool displayable: ((["SUCCESS"].includes(node.globalStatus) && node.chunks.count > 0) || !node.isComputable)
+                                color: displayable ? palette.text : Qt.darker(palette.text, 1.82)
 
                                 ToolTip {
                                     id: nodeImageOutputTooltip
                                     parent: header
                                     visible: nodeImageOutputMA.containsMouse && nodeImageOutput.visible
                                     text: {
                                         if (node.hasImageOutput && !node.has3DOutput)
-                                            return "Double-click on this node to load its outputs in the Image Viewer."
+                                            return nodeImageOutput.displayable ? "Double-click on this node to load its outputs in the Image Viewer." : "This node has an image output."
                                         else if (node.has3DOutput && !node.hasImageOutput)
-                                            return "Double-click on this node to load its outputs in the 3D Viewer."
+                                            return nodeImageOutput.displayable ? "Double-click on this node to load its outputs in the 3D Viewer." : "This node has a 3D output."
                                         else  // Handle case where a node might have both 2D and 3D outputs
-                                            return "Double-click on this node to load its outputs in the Image or 3D Viewer."
+                                            return nodeImageOutput.displayable ? "Double-click on this node to load its outputs in the Image or 3D Viewer." : "This node has an image and a 3D output."
```

Comment:
The messages should be reformulated: a node can have several 2D and/or 3D outputs, so it'd be better to have something like "This node has image outputs." / "This node has 3D outputs." / "This node has image and 3D outputs."